www/config.yml

```diff
@@ -1043,3 +1043,14 @@ chapters:
       location:
             latitude: 7.6559
             longitude: 11.1517
+
+    - name: Doboj, BiH
+        meetup_id: 281140334
```

Comment:
In a yml file, spacing and indenting matters. Please change the remove 2 spaces before (meetup_id, website, ...) so that the title lines up in with name.

Also, please move the Doboj section so that it is in alphabetical order (just after Denver, Colorado).

If you have any questions, please let us know. Thanks :smile: